src/Pos/Communication/Limits.hs

```diff
@@ -209,9 +228,9 @@ instance MessageLimited (DataMsg GtMsgContents) where
     getMsgLenLimit _ =
         return $ each %~ fmap DataMsg $
             ( mcCommitmentMsgLenLimit
-            , noLimit
+            , NoLimit
```

Comment:
> 2.1. If we are dealing with type which has a limited size by structure (e. g. Word8), why can't we just use this limit?

> I expected it to be caught by tests

Well yep, actually I use constant everywhere where possible. And `Limit` gets useful where limit depends on parameters, and without this type I would need to write complex formula.

> But suppose that later someone adds something similar (wrapper over ByteString which must be no more than 26 bytes) and implements Bi instance so that it puts ByteString as usual (i. e. length + contents) and gets it in a similar way, but also checks that size is 26 bytes after parsing. In this case bad things can happen.

Yes, now I see